backend/src/middleware/handleHtmlContent/handleContentData.spec.js

```diff
@@ -68,7 +67,7 @@ describe('currentUser { notifications }', () => {
           'Hey <a class="mention" href="/profile/you/al-capone">@al-capone</a> how do you do?'
 
         beforeEach(async () => {
-          const createPostMutation = gql`
+          const createPostMutation = `
```

Comment:
Can you explain why you removed this?
I think some really like the syntax highlighting this provides and while it might not provide any benefit for you using `vim`, I'm curious if you believe there is any benefit to removing it since some actually really like it.

Reply:
Yeah, I as an example … 😫😉

Reply:
I didn't want to remove it. But `gql` is not for syntax highlighting. It is parsing the string and returns an object. If you we use the parsed object in our tests, apollo complains about sending a parsed object instead of a string. Apparently `graphql-yoga` was indifferent about this.

Reply:
By the way, vim also does the syntax highlighting. So I was benefiting from it, too. :wink: